scripts/foglamp

```diff
@@ -22,7 +22,7 @@ set -e
 #
 # This is the startup script for foglamp
 #
-USAGE="Usage: `basename ${0}` {start|stop|status|reset|kill|help|version}"
+USAGE="Usage: `basename ${0}` {start|start --safe-mode|stop|status|reset|kill|help|version}"
```

Comment:
Seems that only "--" is prefixed to safe-mode only.

Reply:
JIRA has asked for `--safe_mode` and I only replaced underscore with hyphen i.e `--safe-mode `

Reply:
@amarendra-dianomic  Yeah. As it is a  flag with start, hence `start --safe-mode`